internal/repository/rsl.go

```diff
@@ -187,3 +188,31 @@ func (r *Repository) isDuplicateEntry(refName string, targetID plumbing.Hash) (b
 
 	return latestUnskippedEntry.TargetID == targetID, nil
 }
+
+func LogRSLEntries(ctx context.Context, repo *Repository) ([]*rsl.ReferenceEntry, map[plumbing.Hash][]*rsl.AnnotationEntry, [][]*policy.DelegationWithDepth, error) {
+	firstEntry, _, err := rsl.GetFirstEntry(repo.r)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	lastEntry, _, err := rsl.GetLatestReferenceEntryForRef(repo.r, policy.PolicyRef)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
+	entries, annotationMap, err := rsl.GetReferenceEntriesInRangeForRef(repo.r, firstEntry.ID, lastEntry.ID, policy.PolicyRef)
```

Comment:
The order of entries should be reversed, the log should place the most recent entries at the top, if we're aligning with git log's behavior.

Reply:
Fixed